examples/wms-custom-tilegrid-512x256.js

```diff
@@ -25,7 +25,6 @@ var layers = [
     source: new ol.source.MapQuest({layer: 'sat'})
   }),
   new ol.layer.Tile({
-    extent: [-13884991, 2870341, -7455066, 6338219],
```

Comment:
Yes I do. Both cause tiles that intersect the extent to be loaded. But setting the layer extent clips the data at the extent boundary. Especially for WMS, where extents advertised in GetCapabilities are usually not aligned to the tile grid, there is a problem: Label placement is optimized for map/tile boundaries, so you'll get clipped labels when you clip at the extent boundary. But when you use the full tiles that intersect set the tile grid extent, you get nicely placed labels. @bartvde can provide a screenshot that shows the problem.